sdcm/cluster.py

```diff
@@ -3590,6 +3592,14 @@ def remove_sct_metrics(metric):
             scrape_configs.append(dict(job_name="stress_metrics", honor_labels=True,
                                        static_configs=[dict(targets=loader_targets_list)]))
 
+            if cloud_prom_bearer_token:
+                cloud_prom_path = self.params.get('cloud_prom_path', None)
+                scrape_configs.append(dict(job_name="scylla_cloud_cluster", honor_labels=True, scrape_interval='15s',
+                                           metrics_path=cloud_prom_path, scheme='https',
+                                           params={'match[]': ['{job=~".+"}']},
+                                           bearer_token=cloud_prom_bearer_token,
+                                           static_configs=[dict(targets=['us-east-1.aws.metrics.lab.dbaas.scyop.net'])]))
```

Comment:
maybe this should go into configuration ? and not hard-coded ?

Reply:
let's move to siren-tests